pkg/cmd/server/api/v1/types.go

```diff
@@ -377,6 +377,9 @@ type AssetConfig struct {
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL"`
 
+	// CLIDownloadPublicURLs holds links to download client tools, where key is the platform name and value is the download URL (optional)
```

Comment:
@liggitt we've talked with @brenton and @danmcp about specifically letting administrators put in their own download links so they can host `oc` themselves, so this is more than just per product config, its really administrative config